Makefile

```diff
@@ -82,14 +83,13 @@ lib-clean:
 	rm -rf .hypothesis
 
 lib-dep:
+	pip install --upgrade pip && \
 	pip install \
 		-r requirements.txt \
 		-r requirements-testing.txt \
 		-r requirements-release.txt \
-		-r requirements-tunnel.txt
-
-lib-scm-version:
-	@echo "version = '$(VERSION)'" > proxy/common/_scm_version.py
+		-r requirements-tunnel.txt \
+		-r docs/requirements.txt
```

Comment:
It's a bad idea to mix up test and docs requirements. They should be separate.

Reply:
Agreed.  This should be part of `make lib-doc` itself